app/collections/Products.coffee

```diff
@@ -13,3 +13,9 @@ module.exports = class Products extends CocoCollection
       return countrySpecificProduct
     else
       return @findWhere { name: 'basic_subscription' }
+
+  getLifetimeSubscriptionForUser: (user) ->
+    if countrySpecificProduct = @findWhere { name: "#{user?.get('country')}_lifetime_subscription" }
```

Comment:
I strongly prefer not using assignment in `if` conditions; they're easy to misread.

```
countrySpecificProduct = @findWhere { name: "#{user?.get('country')}_lifetime_subscription" }
if countrySpecificProduct
  ...
```